addons/survey/models/survey_question.py

```diff
@@ -747,6 +771,17 @@ def _check_question_not_empty(self):
             if not bool(label.question_id) != bool(label.matrix_question_id):
                 raise ValidationError(_("A label must be attached to only one question."))
 
+    @api.constrains('value', 'value_image')
+    def _check_value(self):
+        question_with_empty_labels = self.env['survey.question']
+        for label in self:
+            if not label.value and not label.value_image:
+                question_with_empty_labels |= label.question_id
```

Comment:
There is already a confusion about answers as being question answers or user_input(_lines) in this module, let's not add another term ^^

```suggestion
        questions_with_empty_valued_answers = self.filtered(lambda a: not a.value and not a.value_image).question_id
```